sycl/doc/extensions/Avoid_event_creation/avoid_event_creation.asciidoc

```diff
@@ -0,0 +1,144 @@
+= Avoid event creation

```

Comment:
This should be the name of the extension, typically something like `SYCL_EXT_ONEAPI_NAME`

Reply:
Bikeshedding: I think "avoid event creation" is a bad name, because "avoid" sounds like a weak suggestion (i.e. "the runtime should avoid creating events if it is able to do so").  The semantics you propose here are much stronger than that -- my interpretation is closer to "the runtime should never create events, even if a developer attempts to use a feature that would require them").

Suggestions:
- `SYCL_EXT_ONEAPI_DISCARD_EVENTS` (implies any events that would be created are discarded instead)
- `SYCL_EXT_ONEAPI_DISABLE_EVENT_CREATION` (implies events are not created at all)
- `SYCL_EXT_ONEAPI_DISABLE_EVENTS` (implies all functionality related to events is disabled)

My preference is "discard events" or "disable events" -- they're both 6 characters shorter than "avoid event creation", and more aligned with my understanding of the proposal.

Reply:
ok, we can change the name, but for now, I changed the name to SYCL_EXT_ONEAPI_DISCARD_EVENTS